src/Resources/config/app/config.yml

```diff
@@ -7,35 +7,6 @@ sylius_refund:
         allowed_files:
             - '%env(default:default_logo_file:resolve:SYLIUS_REFUND_LOGO_FILE)%'
 
-sylius_resource:
```

Comment:
As a way of customizing resources definitions will change, you should probably open this PR against 2.0 branch with a proper note to UPGRADE-2.0 file